partner_communication_switzerland/migrations/12.0.1.0.4/post-migration.py

```diff
@@ -0,0 +1,9 @@
+from openupgradelib import openupgrade
+
+
+def migrate(cr, installed_version):
+    if not installed_version:
+        return
+
+    openupgrade.load_xml(cr, 'partner_communication_switzerland',
+                         'data/sponsorship_communications_cron.xml')
```

Comment:
This is not needed because you created new records with new xml_ids that will be loaded even if noupdate is set.